src/core/Math.gen.tsx

```diff
@@ -12,6 +12,11 @@ export const sq: (_1:number) => number = MathBS.sq;
 
 export const sqf: (_1:number) => number = MathBS.sqf;
 
+export const pow: (_1:{ readonly base: number; readonly exp: number }) => number = function (Arg1: any) {
```

Comment:
You can ignore, the `.bs.js` and `.gen.tsx` files in these diffs since they're autogenerated by BuckleScript / `genType`.